index.html

```diff
@@ -1017,6 +1017,206 @@ <h3>Interface Definition</h3>
 QuicTransportBase includes WebTransport;</pre>
       </div>
     </section>
+    <section>
+      <h2>Methods</h2>
+      <dl data-link-for="QuicTransportBase" data-dfn-for="QuicTransportBase" class=
+      "methods">
+       <dt><dfn><code>getStats</code></dfn></dt>
+        <dd>
+        <p>Gathers stats for this <code><a>QuicTransportBase</a></code>'s QUIC connection and reports
+        the result asynchronously.</p>
+          <p>When <code>getStats</code> is called, the user agent <em class="rfc2119" title="MUST">MUST</em>
+          run the following steps:</p>
+          <ol>
+            <li>Let <var>transport</var> be the <code><a>QuicTransportBase</a></code>
+            on which <code>getStats</code> is invoked.</li>
+            <li>If <var>transport</var>'s <a>[[\WebTransportState]]</a> is <code>"new"</code>
+            then abort these steps.</li>
+            <li>Return <var>p</var> and continue the following steps in the background.</li>
+            <ol>
+              <li>Gather the stats from the underlying QUIC connection.</li>
+              <li>Once stats have been gathered, resolve <var>p</var> with the
+              <code><a>QuicTransportStats</a></code> object, representing the gathered
+              stats</li>
+            </ol>
+          </ol>
+          <div>
+            <em>No parameters.</em>
+          </div>
+          <div>
+            <em>Return type:</em> <code><a>QuicTransportStats</a></code>
+          </div>
+        </dd>
+    </section>
+    <section id="quictransportstats*">
+      <h3><dfn>QuicTransportStats</dfn> Dictionary</h3>
+      <p>The <code>QuicTransportStats</code> dictionary includes information
+      QUIC connection level stats.
+      <div>
+        <pre class="idl">
+dictionary QuicTransportStats {
```

Comment:
This is a rich set of stats, and it's unclear to me how much of them we want.  I assume this is roughly the QuicConnectionStats object that gQUIC exposes.  I feel like we should be more conservative about what we expose, since it's not necessarily the case that other browsers' QUIC implementation tracks all of those.

Reply:
Yes you're correct, this is what I called out in the first PR comment - these are roughly the stats from the origin trial, which comes from gQUIC stats, & I'm happy to remove/edit. We could just keep a minimum for now and add later. WDYT of the following?

```
DOMHighResTimeStamp timestamp;
unsigned long long bytesSent;
unsigned long numOutgoingStreamsCreated;
unsigned long numIncomingStreamsCreated;
unsigned long long bytesReceived;
unsigned long estimatedBandwidthBps;
unsigned long long minRttUs;
unsigned long numReceivedDatagramsDropped;
```

Reply:
I updated to include a subset of the original PR. I'm resolving this conversation, please comment if you think there are others that should be removed. Thanks!